addons/website/models/website.py

```diff
@@ -386,6 +386,10 @@ def configurator_skip(self):
         website = self.get_current_website()
         website.configurator_done = True
 
+    @api.model
+    def configurator_send_iap(self, unmatch_term):
```

Comment:
`configurator_send_iap` doesn't hint anything, let's rename it to `configurator_missing_industry`